scenarios.js

```diff
@@ -1,25 +1,29 @@
 SCENARIO_DEFINITIONS =
-    [   { name: "#1 Black Barrow"
+    [   { number: 1

```

Comment:
Shall we do that? We're still saving the number in the name, although it looked more human-readable this way. And, if this is the case, then we have to include an empty scenario 55.